src/libdredd/src/mutation_remove_statement.cc

```diff
@@ -43,9 +43,10 @@ void MutationRemoveStatement::Apply(
       clang::tok::TokenKind::semi, ast_context);
 
   bool result = rewriter.ReplaceText(
-      source_range, "if (!__dredd_enabled_mutation(" +
-                        std::to_string(mutation_id) + ")) { " +
-                        rewriter.getRewrittenText(source_range) + " }");
+      source_range,
+      "if (!__dredd_enabled_mutation(" +
+          std::to_string(mutation_id - first_mutation_id_in_file) + ")) { " +
```

Comment:
It may be worth mentioning again that this is to convert to the file specific mutation ID.

Reply:
Done.